data/data/azure/vnet/public-lb.tf

```diff
@@ -144,7 +144,8 @@ resource "azurerm_lb_rule" "public_lb_rule_api_internal_v6" {
 }
 
 resource "azurerm_lb_outbound_rule" "public_lb_outbound_rule_v4" {
-  count = var.use_ipv4 && var.azure_private && ! var.azure_outbound_user_defined_routing ? 1 : 0
+  #count = var.use_ipv4 && var.azure_private && ! var.azure_outbound_user_defined_routing ? 1 : 0
+  count = 0
```

Comment:
What does disabling this do?

Reply:
I just wanted to make sure no outbound traffic was going through the LB but through the NAT instead. Probably not needed since NAT gateway takes precedence for outbound: https://learn.microsoft.com/en-us/azure/virtual-network/nat-gateway/nat-gateway-resource#nat-and-vm-with-a-standard-public-load-balancer